src/lib.rs

```diff
@@ -197,6 +205,8 @@ pub mod pallet {
 		CannotDeleteSupersig,
```

Comment:
Rename `CannotDeleteSupersig` so it explain the locked tokens thing.
Better having the name be explicit, than having people read the doc